Library/Homebrew/formula_installer.rb

```diff
@@ -1001,14 +1001,48 @@ def link(keg)
     @show_summary_heading = true
   end
 
+  sig { params(service_hash: Hash).String }
+  def generate_plist(service_hash)
+    items = ["<key>Label</key><string>#{formula.plist_name}</string>"]
```

Comment:
Might be nice to use the `REXML` module to build the XML?

Reply:
We have already vendored the `Plist` gem. You should just be able to call `#to_plist` on a Hash.

Reply:
Even better 🎉